libsigner/src/events.rs

```diff
@@ -209,27 +217,45 @@ impl EventReceiver for StackerDBEventReceiver {
                     &request.method(),
                 )));
             }
-            if request.url() != "/stackerdb_chunks" {
-                let url = request.url().to_string();
+            if request.url() == "/stackerdb_chunks" {
+                debug!("Got stackerdb_chunks event");
+                let mut body = String::new();
+                request
+                    .as_reader()
+                    .read_to_string(&mut body)
+                    .expect("failed to read body");
 
```

Comment:
I think you should respond `200` as soon as you've read body into the string. The `stacks-node` will stall progress until it gets a 200 response, and if you exit this function early due to error handling, the `stacks-node` may end up stalling entirely.

Reply:
Done!